src/group_theory/schur_zassenhaus.lean

```diff
@@ -78,29 +78,27 @@ begin
   exact self_eq_mul_right.mpr ((quotient_group.eq_one_iff _).mpr h.2),
 end
 
-variables [fintype H]
-
-lemma eq_one_of_smul_eq_one (hH : nat.coprime (fintype.card H) H.index)
+lemma eq_one_of_smul_eq_one (hH : nat.coprime (nat.card H) H.index)
```

Comment:
LGTM, just noting that dot notations could also be used in this file.

maintainer merge